configuration/configuration-mainnet.yaml

```diff
@@ -167,7 +167,7 @@ TraceLocalTxSubmissionServer: False
 TraceMempool: True
 
 # Trace Mux Events
-TraceMux: False
+TraceMux: True
```

Comment:
I'm not convinced. Turning this on here but then filtering out everything is still expensive since this is a high volume tracer right? Have you measured the performance impact?

Reply:
I haven't yet; We can split the mux tracer into two parts to avoid a performance regression.

Reply:
I couldn't compile `cardano-node` yestrday due to a glitch; now looking at the disk usage: with the `mux` tracer off it's 20% faster (500KB/s -> 600KB/s) with a similar network performance measured by RX rate on the interface.  We'll need to split the tracer to more useful logging.